src/components/rangeselector/button_attributes.js

```diff
@@ -0,0 +1,44 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+
+module.exports = {
+    step: {
+        valType: 'enumerated',
+        role: 'info',
+        values: ['month', 'year', 'day', 'all'],
+        dflt: 'month',
+        description: [
+            ''
+        ].join(' ')
+    },
+    stepmode: {
+        valType: 'enumerated',
+        role: 'info',
+        values: ['backward', 'to date'],
```

Comment:
Should we go for `to date` or `todate`? Other attributes (`tozeroy` for example) are stripped of whitespace.